libs/data-mapper/src/lib/components/configPanel/AddOrUpdateSchemaView.tsx

```diff
@@ -208,13 +208,16 @@ export const AddOrUpdateSchemaView = ({
       )}
 
       {uploadType === UploadSchemaTypes.SelectFrom && (
-        <Dropdown
+        <VirtualizedComboBox
```

Comment:
Could we also get the search functionality to filter? https://react.fluentui.dev/?path=/docs/components-combobox--default#filtering